saturn-datepicker/src/datepicker/calendar-body.ts

```diff
@@ -112,6 +118,8 @@ export class SatCalendarBody implements OnChanges {
   /** Width of an individual cell. */
   _cellWidth: string;
 
+  _cellOver: number;
+
```

Comment:
Please, add description of variable. And add "private" access, due to this property isn't used in template